lib/search/query.rb

```diff
@@ -61,39 +90,13 @@ def run_spell_checks(search_params)
       return unless suggestion_blocklist.should_correct?(search_params.query)
 
       query = {
-        size: 0,
-        suggest: QueryComponents::Suggest.new(search_params).payload
+          size: 0,
```

Comment:
Not sure why the linter didn't pick up extra whitespace 🤔 It'd be good to remove this, as it will preserve the `blame` trail better.